### PR TITLE
Update steps in IG.

### PR DESCRIPTION
**Pull request by yhontyk**
> Read our [Contribution guide](https://github.com/eclipse/che-docs/blob/master/CONTRIBUTING.adoc) before submitting a PR.

### What does this PR do?
Updates the working in installation guide steps.

### What issues does this PR fix or reference?
https://issues.redhat.com/browse/RHDEVDOCS-2187

### Specify the version of the product this PR applies to.
7.18 and later

### PR Checklist

As the author of this Pull Request I made sure that:

- [ ] `vale` has been run successufully against the PR branch
- [ ] Link checker has been run successfully against the PR branch
- [ ] Documentation describes a scenario that is already covered by QE tests, otherwise an issue has been created and acknowledged by Che QE team
- [ ] Changed article references are updated where they are used (or a redirect has been set up on the docs side):
    - [ ] Dashboard [branding.constant.ts](https://github.com/eclipse/che-dashboard/blob/master/src/components/branding/branding.constant.ts) + [product.json](https://github.com/eclipse/che-dashboard/blob/master/src/assets/branding/product.json)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)

### CODE REVIEW

**Comment by yhontyk** [comment]
Screenshot from 2020-09-14 15-59-12
 To clarify - this is how the GUI actually looks lile.

File: modules/installation-guide/partials/proc_installing-che-using-the-che-operator-in-openshift-4-web-console.adoc [hunk 1]
```diff
@@ -45,10 +45,10 @@ Operators are a method of packaging, deploying, and managing a Kubernetes applic
...
 . In the *{prod}* pop-up window, click the btn:[Install] button .
 
-. On the *Install Operator* screen, choose following options and validate:
+. On the *Install Operator* screen, specify the following options:
 +
 * *Installation mode*: *A specific {orch-namespace} on the cluster*.
-* *Installed Namespace*: `{prod-short}`.
+* *Installed Namespace*: *Pick an existing namespace* -> `{prod-short}`.
 
 . Click the *Install* button.
```
Comment (rkratky) [inline modules/installation-guide/partials/proc_installing-che-using-the-che-operator-in-openshift-4-web-console.adoc]:
```suggestion
. Click the btn:[Install] button. 
```

File: modules/installation-guide/partials/proc_installing-che-using-the-che-operator-in-openshift-4-web-console.adoc [hunk 2]
```diff
@@ -45,10 +45,10 @@ Operators are a method of packaging, deploying, and managing a Kubernetes applic
 
 . In the *{prod}* pop-up window, click the btn:[Install] button .
 
-. On the *Install Operator* screen, choose following options and validate:
+. On the *Install Operator* screen, specify the following options:
 +
 * *Installation mode*: *A specific {orch-namespace} on the cluster*.
-* *Installed Namespace*: `{prod-short}`.
+* *Installed Namespace*: *Pick an existing namespace* -> `{prod-short}`.
```
Comment (rkratky) [inline modules/installation-guide/partials/proc_installing-che-using-the-che-operator-in-openshift-4-web-console.adoc]: Just to make sure... shouldn't the value be `{prod-id-short}` here?

File: modules/installation-guide/partials/proc_installing-the-che-operator.adoc [hunk 3]
```diff
@@ -7,6 +7,10 @@
 
 {prod} Operator provides with all the resources for running {prod-short} such as PostgreSQL, Keycloak, Registries and the {prod-short} server, as well as configures all these services.
```
Comment (rkratky) [inline modules/installation-guide/partials/proc_installing-the-che-operator.adoc]:
```suggestion
{prod} Operator provides all the resources for running {prod-short}, such as PostgreSQL, {identity-provider}, image registries, and the {prod-short} server, and also configures all these services.
```

File: modules/installation-guide/partials/proc_installing-the-che-operator.adoc [hunk 4]
```diff
@@ -15,11 +19,10 @@
 
 . In the *{prod}* pop-up window, click the btn:[Install] button.
 
-. On the *Install Operator* screen, choose following options and validate:
+. On the *Install Operator* screen, specify the following options:
 +
 * *Installation mode*: *A specific {orch-namespace} on the cluster*
-* *Installed Namespace*: `{prod-short}`
-//(Operator recommended namespace: ProjectPRopenshift-workspaces) ?
+* *Installed Namespace*: *Pick an existing namespace* -> `{prod-short}`
```
Comment (rkratky) [inline modules/installation-guide/partials/proc_installing-the-che-operator.adoc]: Just to make sure... shouldn't the value be `{prod-id-short}` here?

**Review by rkratky — APPROVED** [review]
Thanks, @boczkowska. Preapproving, but have a look at my questions/suggestions, please.

File: modules/installation-guide/partials/proc_installing-che-using-the-che-operator-in-openshift-4-web-console.adoc (hunk 2 above)
Comment (themr0c) [reply modules/installation-guide/partials/proc_installing-che-using-the-che-operator-in-openshift-4-web-console.adoc]:
```suggestion
* *Installed Namespace*: *Pick an existing namespace* -> `{prod-namespace}`.
```

File: modules/installation-guide/partials/proc_installing-the-che-operator.adoc (hunk 4 above)
Comment (themr0c) [inline modules/installation-guide/partials/proc_installing-the-che-operator.adoc]:
```suggestion
* *Installed Namespace*: *Pick an existing namespace* -> `{prod-namespace}`
```

File: modules/installation-guide/partials/proc_installing-che-using-the-che-operator-in-openshift-4-web-console.adoc (hunk 2 above)
Comment (yhontyk) [reply modules/installation-guide/partials/proc_installing-che-using-the-che-operator-in-openshift-4-web-console.adoc]: @rkratky in downstream it is defined as `:prod-namespace: workspaces` so it is incorrect here, see the screenshot I posed in the PR.

File: modules/installation-guide/partials/proc_installing-the-che-operator.adoc (hunk 4 above)
Comment (yhontyk) [reply modules/installation-guide/partials/proc_installing-the-che-operator.adoc]: It is not, see my commend above.

File: modules/installation-guide/partials/proc_installing-che-using-the-che-operator-in-openshift-4-web-console.adoc (hunk 2 above)
Comment (themr0c) [reply modules/installation-guide/partials/proc_installing-che-using-the-che-operator-in-openshift-4-web-console.adoc]: We do not pick an existing namespace, we pick the namespace we created befro in the procedure (step 4).

Comment (yhontyk) [reply modules/installation-guide/partials/proc_installing-che-using-the-che-operator-in-openshift-4-web-console.adoc]: Hmm, okay, my mistake.

**Pull request merged**